### ISSUE TITLE
2024 -2025 Mobilisation: WA Herbarium (Biosec)

### ISSUE THREAD

**Issue by rosemaryjoconnor** [AtlasOfLivingAustralia/data-management#1061]

**Comment by rosemaryjoconnor** [comment]
**08/07/2024** - initial meeting

Discussed regular updates, sample data and how images to be provided.
No problems envisaged, all looking very straightforward.

First steps:
- [ ] ALA to provide multimedia template

Perth will provide some sample data/images for testing in databox asap

**Comment by rosemaryjoconnor** [comment]
**12/08/2024**
- Templates sent to JPB

**Comment by rosemaryjoconnor** [comment]
**10/09/2024**
- JPB provide initial multimedia template for review

**Comment by rosemaryjoconnor** [comment]
**13/09/2024**

Not having much success with the sample images in databox so far.
Will continue with it next week

**Comment by rosemaryjoconnor** [comment]
**16/09/2024**

- Issue with images found. The images in the mutlimedia file are for new occurrence records that are not currently in ALA. I had thought they would be. dr15863 is fairly large so I hadn't looked at the occurrences closely. 
- Have emailed JPB re getting occurrence records for these for test only or provision of some images for existing records.

**Comment by nielsklazenga** [comment]
@rosemaryjoconnor, it turns out that all `catalogNumber`s (which is their unique field) have changed in last week's DwCA. Would you like me to load the previous version in Databox, so that you can see if it works with that?

**Comment by rosemaryjoconnor** [comment]
@nielsklazenga no. I don't want to load any images even in test if the unique field may change. Testing of images can wait until WA Herbarium is sorted out

**Comment by rosemaryjoconnor** [comment]
**31/10/2024**

- OccurrenceIDs and catalogNumbers were in error. 
- NK has dealt with that and new data will be supplied to us
- Currently loading Prod to Databox

**Comment by rosemaryjoconnor** [comment]
**13/11/2024**
Issues with catalogNumbers fixed by Herbarium
Sample dataset with multimedia provided
Loaded sample dataset to databox
Images loaded successfully
Emailed re success 14/11/2024

**Comment by rosemaryjoconnor** [comment]
**19/11/2024**

- Set reminder to contact for update Dec 13th

**Comment by rosemaryjoconnor** [comment]
**11/04/2025**

JPB emailed

Data set provided via Sharepoint

Here’s a quick breakdown of the folders and the data they contain:

**new records added to PERTH collection**

- Occurrence csv contains the records of weedy/invasive species that we’ve added to our database since the start of the project (473 records).
- These records are already being delivered to ALA, this is only to show what we’ve added so far. Let me know if there’s a better way to convey this information.

**weeds_imaged**

- Occurrence and multimedia csv files that contain the data for the specimens that we’ve imaged so far as part of the project.
- There are 16,538 images of 16,132 specimens (some specimens required multiple image captures).

**non-weeds_imaged**

- Not part of our funding, but I’ve compiled occurrence and multimedia csv files for everything else that PERTH has imaged and is available online. Wondering if you’d be interested in making these available through ALA too?
- There are 79,215 images of 75,387 specimens (some specimens required multiple image captures).

Checking with Niels
